### PR TITLE
docs(PPDSC-2269): Tooltip documentation

### PR DESCRIPTION
**Pull request by Srinivasan-Ramasamy**
PPDSC-2269

**What**

1. Tooltip documentation page UI - http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2269-tooltip-documentation/components/tooltip/

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [X] Updated relevant documentation

**I have tested manually:**
- [X] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [X] The screen reader reads and flows through the elements as expected.
- [X] There are no new errors in the browser console coming from this PR.
- [X] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

image

**Who should review this PR:**

**How to test:**

URL: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2269-tooltip-documentation/components/tooltip/

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2269-tooltip-documentation
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2269-tooltip-documentation/storybook

**Review by baburay23 — DISMISSED** [review]

File: site/pages/components/tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+            },
+            {
+              attribute: 'panel.typographyPreset',
+              type: 'MQ<string>',
+              default: 'utilityLabel010',
+              description:
+                'Overrides the typographyPreset applied to the Tooltip panel.',
+            },
+            {
+              attribute: 'panel.paddingBlockStart',
```
Comment (Xin00163) [inline site/pages/components/tooltip.tsx:568]: Missing `panel.paddingBlockEnd" and some details for `type` and `description`

**Review by Xin00163 — COMMENTED** [review]
A small change

File: site/pages/components/tooltip.tsx (hunk 1 above)
Comment (Srinivasan-Ramasamy) [reply site/pages/components/tooltip.tsx:568]: Thanks @Xin00163 , updated the changes. 

and no details on type and description in doc-> https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/3942350849/Tooltip+Component+-+Web+Documentation

Comment (Srinivasan-Ramasamy) [reply site/pages/components/tooltip.tsx:568]: Please suggest.

Comment (Xin00163) [reply site/pages/components/tooltip.tsx:568]: Hey @Srinivasan-Ramasamy , those are normally autogenerated so Design probably didn't put it in the spec. In this case, paddingBlock and paddingInline have defaults, so we can't use `commonLogicalProps`. The type and description should be consistent with what's in [here](https://github.com/newscorp-ghfb/newskit/blob/main/site/components/component-api/common-logical-props.tsx)

Comment (Srinivasan-Ramasamy) [reply site/pages/components/tooltip.tsx:568]: Thanks a lot @Xin00163 for the details, I've missed this one. Done the changes :+1:

**Review by Xin00163 — APPROVED** [review]

**Comment by Srinivasan-Ramasamy** [comment]
Thanks @Xin00163  and @baburay23 for reviewing the code :+1:

**Review by baburay23 — APPROVED** [review]

**Pull request merged**